### PR TITLE
made a better embed - more to come 

### PR DESCRIPTION
**Pull request by wadder12**
- added a better embed for now 

(let me help with the bot a lot more) lol

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,4 +0,0 @@
-CONF_DISCORD_TOKEN=
```
Comment (mrjvs) [inline .env.example:1]: Please don't delete this file

**Review by mrjvs — CHANGES_REQUESTED** [review]
Preliminary review.

@castdrian could you review the rest, Or want me to handle it?

**Comment by castdrian** [comment]
> Preliminary review.
> 
> @castdrian could you review the rest, Or want me to handle it?

I'll be happy to

File: src/commands/status.ts [hunk 2]
```diff
@@ -1,54 +1,68 @@
...
+
+
+
+  public override registerApplicationCommands(registry: Command.Registry): void {
     registry.registerChatInputCommand((builder) =>
-      builder //
+      builder
         .setName('status')
-        .setDescription('movie-web status'),
+        .setDescription('Check the status of Movie-Web sites'),
```
Comment (castdrian) [inline src/commands/status.ts:65]: unnecessary change, breaks the consitent casing of the other strings as well

File: src/commands/status.ts [hunk 3]
```diff
@@ -1,54 +1,68 @@
...
+        const response = await fetch(url);
+        return `${response.ok ? `🟢 UP` : '🔴 DOWN'} - [${url}](${url})`;
+      } catch (error) {
+        console.error('Error checking URL:', url, error);
+        return `🔴 DOWN (Error) - [${url}](${url})`;
+      }
+    }));
+  }
+
+  private createEmbeds(statusResults: string[]): EmbedBuilder[] {
```
Comment (castdrian) [inline src/commands/status.ts:37]: there are no magic numbers involved so I don't see any particular reason to prefer the builder over a plain object, however the main concern is the unnecessary function rather than the builder

File: src/commands/status.ts [hunk 4]
```diff
@@ -1,54 +1,68 @@
...
+    const exampleEmbed = new EmbedBuilder()
+      .setColor(0x0099FF)
+      .setTitle('Movie-Web Status')
```
Comment (castdrian) [inline src/commands/status.ts:40]: this breaks the consistent casing across the strings

File: src/commands/status.ts [hunk 5]
```diff
@@ -1,54 +1,68 @@
...
+      .setURL('https://movie-web.app') // Replace with your website or relevant link
```
Comment (castdrian) [inline src/commands/status.ts:41]: if you use chatgpt at least get rid of the comments it adds please

File: src/commands/status.ts [hunk 6]
```diff
@@ -1,54 +1,68 @@
...
+      .setAuthor({ 
+        name: 'Movie-Web Bot',
```
Comment (castdrian) [inline src/commands/status.ts:43]: string casing as above

File: src/commands/status.ts [hunk 7]
```diff
@@ -1,54 +1,68 @@
...
+  }
+
+  private createEmbeds(statusResults: string[]): EmbedBuilder[] {
+    const exampleEmbed = new EmbedBuilder()
+      .setColor(0x0099FF)
+      .setTitle('Movie-Web Status')
+      .setURL('https://movie-web.app') // Replace with your website or relevant link
+      .setAuthor({ 
+        name: 'Movie-Web Bot', 
+        iconURL: 'https://avatars.githubusercontent.com/u/121455091?s=200&v=4', // Replace with your bot's icon URL
```
Comment (castdrian) [inline src/commands/status.ts:44]: this url should be a constant somewhere if it's necessary at all

File: src/commands/status.ts [hunk 8]
```diff
@@ -1,54 +1,68 @@
...
+        url: 'https://movie-web.app' // Replace with your website or relevant link
+      })
+      .setDescription('Current status of Movie-Web URLs')
```
Comment (castdrian) [inline src/commands/status.ts:47]: again the casing of movie-web is generally lowercase

File: src/commands/status.ts [hunk 9]
```diff
@@ -1,54 +1,68 @@
...
+      .setThumbnail(this.container.client.user?.displayAvatarURL() ?? config.mwIconUrl)
+      .addFields(statusResults.map(status => ({ name: '\u200B', value: status })))
+      .setTimestamp()
+      .setFooter({ 
+        text: 'Status checked at', 
+        iconURL: 'https://avatars.githubusercontent.com/u/121455091?s=200&v=4' // Optional: Footer icon URL
```
Comment (castdrian) [inline src/commands/status.ts:53]: chatgpt comment again

File: src/commands/status.ts [hunk 10]
```diff
@@ -1,54 +1,68 @@
...
+  public override async chatInputRun(interaction: CommandInteraction): Promise<void> {
     try {
       await interaction.deferReply();
+      const statusResults = await this.checkUrlsStatus(mwUrls);
+
+      const embeds = this.createEmbeds(statusResults);
+      // Uncomment the next line if you use components
+      // const components = this.createComponents();
 
-      const description = await Promise.all(
```
Comment (castdrian) [inline src/commands/status.ts:12]: this entire file is not linted and failing lint checks

**Review by castdrian — CHANGES_REQUESTED** [review]
